Xamarin.Forms.Platform.Android/Forms.cs

```diff
@@ -224,7 +224,8 @@ public override double ScalingFactor
 
 			protected override void Dispose(bool disposing)
 			{
-				_formsActivity.ConfigurationChanged -= ConfigurationChanged;
+				if (_formsActivity is IDeviceInfoProvider)
```

Comment:
This `if` block needs to be surrounded by `if (disposing){}` so it doesn't run in the destructor.